keep-mega-alive.sh

```diff
@@ -1,21 +1,46 @@
 #!/bin/sh
 ###
-# Keep-MEGA-Alive v1.1
+# Keep-MEGA-Alive v1.2
 ##########################
 
+exec 4>>~/keep-mega-alive.log
+
 LOGINS=${1:-"mega-logins.csv"}
 
 if ! [ -x "$(command -v mega-version)" ]; then
 	echo 'Error: MEGAcmd is not installed. Get it from https://mega.io/cmd' >&2
 	exit 1
 fi
 
+echo -e "$(date "+%m%d%Y %T") : Starting keep-mega-alive \n" >&4
```

Comment:
Can you change the log time format to a more universal one like [`ISO 8601`](https://www.iso.org/iso-8601-date-and-time-format.html):

```
date +"%Y-%m-%dT%H:%M:%S%:z"
```